src/sentry/static/sentry/app/views/organizationIntegrations/addIntegrationButton.jsx

```diff
@@ -85,21 +91,21 @@ export default class AddIntegrationButton extends React.Component {
 
   render() {
     // eslint-disable-next-line no-unused-vars
-    const {provider, onAddIntegration, ...buttonProps} = this.props;
+    const {provider, onAddIntegration, reinstall, ...buttonProps} = this.props;
+    let buttonText = reinstall ? t('Reinstall') : t('Add') + ' ' + provider.metadata.noun;
 
     return (
       <Tooltip
         disabled={provider.canAdd}
         tooltipOptions={{placement: 'left'}}
-        title={`Integration cannot be added on Sentry. Enable this integration via the ${provider.name} instance.`}
-      >
+        title={`Integration cannot be added on Sentry. Enable this integration via the ${provider.name} instance.`}>
```

Comment:
This is weird, are you using a different version of prettier. Afaik the formatting of multiline jsx elements has the close tag on a new line /cc @billyvg